File: PHP Excel Examples/WorksheetsApiDemo/GetMergedCell/getMergedCells.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "GetMergedCells.xlsx";
$sheetName = "Sheet5";
$folder= "input";
$storage = null;
$result = $apiInstance->getMergedCells($name, $sheetName,$folder, $storage);

File: Python Excel Examples/WorksheetsApiDemo/get_merged_cell/getMergedCells.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "GetMergedCells.xlsx"
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
mergedCellIndex = 0
result = api.get_merged_cells(name, sheet_name, folder=folder, storage=storage)